test/dotnet-new3.UnitTests/Approvals/DotnetNewHelp.CanShowHelp_List.approved.txt

```diff
@@ -0,0 +1,17 @@
+﻿Description:
+  Lists templates containing the specified template name. If no name is specified, lists all templates.
+
+Usage:
+  new3 list [<name>] [options]
+
+Arguments:
+  <name>  If specified, only the templates matching the name will be shown.
+
+Options:
+  --author <author>                      Filters the templates based on the template author.
+  -lang, --language <language>           Filters templates based on language.
+  --type <type>                          Filters templates based on available types. Predefined values are "project" and "item".
+  --tag <tag>                            Filters the templates based on the tag.
+  --columns-all                          Display all columns in the output.
+  --columns <author|language|tags|type>  Specifies the columns to display in the output. 
```

Comment:
random thought not directly part of this rewrite - why `--columns-all` instead of `--columns all`. making `all` just another display option.